stats.go

```diff
@@ -12,6 +12,7 @@ const (
 	StatsRelayProxyHeaderStreamReceived   = "stats.relay-proxy-headerStreamReceived"
 	StatsRelayProxyBlockStreamReceived    = "stats.relay-proxy-blockStreamReceived"
 	StatsRelayProxyGetHeaderExternalRelay = "stats.relay-proxy-getHeader-externalRelay"
+	StatsRelayProxyPerformanceStats       = "stats.relay-proxy-performance-stats"
```

Comment:
```suggestion
	StatsRelayProxyPerformanceStats       = "builder-relay.stats.performance"
```

The reason for this is that our data collection server uses `builder-relay.stats.performance` to map this data entry to our mysql database. So to get this going to the correct table we need to use `builder-relay.stats.performance`